crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/CrownCourtEligibilityService.java

```diff
@@ -75,14 +77,14 @@ boolean hasRequiredCaseTypeAndOutcome(MeansAssessmentRequestDTO assessmentReques
                 caseType == CaseType.EITHER_WAY && magCourtOutcome == MagCourtOutcome.COMMITTED_FOR_TRIAL;
     }
 
+    Stream<Assessment> getPreviousAssessments(RepOrderDTO repOrder) {
```

Comment:
Is there a reason why so many methods in this service class have the more public scope/visibility of `package-private` instead of `private`? In general we should be minimising the scope of methods, variables etc. to the minimum level required.

Reply:
Point taken, I've updated the relevant access modifiers